src/CrashlogFile.ts

```diff
@@ -1,8 +1,25 @@
 import * as path from 'path';
-import type { FileReference } from './interfaces';
+import type { ApplicationVersionCount, CrashReport, FileReference, LocalVariable, Location, RokuHardwarePlatform, StackTraceStep } from './interfaces';
+
+// eslint-disable-next-line sort-imports
+import { util as bscUtil, standardizePath } from 'brighterscript';
 import type { Position } from 'brighterscript';
 import type { Runner } from './Runner';
-import { util as bscUtil } from 'brighterscript';
+import { util } from './util';
+
+// TODO: where to put this
+enum CrashReportSectionType {
+    HardwarePlatform = 'HardwarePlatform',
+    ApplicationVersion = 'ApplicationVersion',
+    StackTrace = 'StackTrace'
+}
+
+const HardwarePlatformHeaderRegex = /\s*count\s+Hardware Platform/;
```

Comment:
The javascript engine is very good at caching regex literals, so you don't need to lift these into their own variables. Can you move these inline with where they're used?